<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="lineEditDigIP"/>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonConnect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labelMin">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="16">
     <widget class="QPlainTextEdit" name="plainTextEditInfos">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLCDNumber" name="lcdNumberMax"/>
    </item>
    <item row="2" column="4">
     <widget class="QLCDNumber" name="lcdNumberMin"/>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="horizontalSliderMin">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="horizontalSliderMax">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="labelMax">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelTiming">
      <property name="text">
       <string>Timing(s)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QPushButton" name="pushButtonStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButtonDisconnect">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QSlider" name="horizontalSliderTiming">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="labelTiming_2">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCliente_produtor_de_dados">
    <property name="title">
     <string>Produtor de dados</string>
    </property>
   </widget>
   <addaction name="menuCliente_produtor_de_dados"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDisconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>discon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>mostraLcdMaximo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>mostraLcdMinimo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTiming</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>mostraTempo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelTiming_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tcpConnect()</slot>
  <slot>discon()</slot>
  <slot>start()</slot>
  <slot>Stop()</slot>
  <slot>mostraLcdMaximo()</slot>
  <slot>mostraLcdMinimo()</slot>
  <slot>mostraTempo()</slot>
 </slots>
</ui>
